From ac8ae3215a28360b9ceec9f90ceb59879da2fcaf Mon Sep 17 00:00:00 2001
From: Alexander Shiyan <shc_work@mail.ru>
Date: Sat, 10 Aug 2013 12:51:50 +0400
Subject: [PATCH 149/161] ARM: dts: i.MX27: Disable AUDMUX in the template

AUDMUX expects additional parameters to be configured correctly,
so turn it off into a template.

Signed-off-by: Alexander Shiyan <shc_work@mail.ru>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx27.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx27.dtsi b/arch/arm/boot/dts/imx27.dtsi
index 76cd89f..c037c22 100644
--- a/arch/arm/boot/dts/imx27.dtsi
+++ b/arch/arm/boot/dts/imx27.dtsi
@@ -300,6 +300,7 @@
 				reg = <0x10016000 0x1000>;
 				clocks = <&clks 0>;
 				clock-names = "audmux";
+				status = "disabled";
 			};
 
 			cspi3: cspi@10017000 {
-- 
1.7.10.4

